src/backend/joanie/tests/core/utils/test_issuers_certificate_and_degree_generate_document.py

```diff
@@ -132,6 +132,15 @@ def test_utils_issuers_generate_document_certificate_document(self):
         self.assertRegex(
             document_text, r"Joanie Cunningham.*Course with attestation.*University X"
         )
+        self.assertRegex(
+            document_text,
+            (
+                r"The current document is not a degree or diploma and "
+                r"does not award credits \(ECTS\)\."
+                r" It does not certify that the learner was registered with University X\."
```

Comment:
Would it be possible to add another organization to your test to see if the filter work as expected ?

Reply:
I updated the test.